src/main/webapp/resources/js/pages/projects/remote/ProjectSynchronizationSettings.jsx

```diff
@@ -0,0 +1,113 @@
+      setRemoteProjectData(remoteProjectSettings);
+    }).catch((message) => {
+      notification.error({ message });
+    });
+  }, []);
+
+  const syncSettings = remoteProjectData === null ? [] : [
+      {
+        title: i18n("ProjectRemoteSettings.lastSync"),
+        desc: (<div>
+          <span>{formatDate({ date: remoteProjectData.lastUpdate })}</span>
+          <br />
+          <Button type="primary"
+                  onClick={() => updateSyncSettings({ forceSync: true })}
+                  disabled={(disableSyncNow ? true : false) ||
+                            (syncNowEnabledStates.includes(remoteProjectData.remoteStatus.syncStatus) ? false : true)}
+                  className="t-sync-now-btn"
+          >
+            {i18n("ProjectRemoteSettings.syncNow")}
+          </Button>
+        </div>)
+      },
+      {
+        title: i18n("ProjectRemoteSettings.remoteConnection"),
+        desc: (<span>{remoteProjectData.remoteAPI.label}<RemoteApiStatus
+          key="status"
+          api={{ id: remoteProjectData.remoteAPI.identifier }}
+        /></span>)
+      },
+      {
+        title: <span>
+                <span>{i18n("ProjectRemoteSettings.syncFrequency")}</span>
+                <HelpPopover
+                  content={<div>{i18n("SyncFrequencySelect.frequency.help")}</div>}
+                />
+               </span>,
+        desc: (
+          <Form initialValues={{
+            frequency: remoteProjectData.projectSyncFrequencies.indexOf(remoteProjectData.projectSyncFrequency),
+          }}>
+            <SyncFrequencySelect
+              onChange={(e) => updateSyncSettings({ projectSyncFrequency: remoteProjectData.projectSyncFrequencies[e] })}
+              labelRequired={false}
+            />
+          </Form>
+        )
+      },
+      {
+        title: i18n("ProjectRemoteSettings.syncUser"),
+        desc: (<div>
+          <span>{remoteProjectData.syncUser.label}</span>
+          <br />
+          { window.TL._USER.identifier !== remoteProjectData.syncUser.identifier ?
+              <Button
+                onClick={() => updateSyncSettings({ changeUser: true })}
+                className="t-become-sync-user-btn"
+              >
+                {i18n("ProjectRemoteSettings.becomeSyncUser")}
+              </Button>
+            : null
+          }
+        </div>)
+      }
+    ];
+
+  // Used to update sync user, sync frequency, and force to sync now
+  function updateSyncSettings({forceSync, changeUser, projectSyncFrequency}) {
+    updateRemoteProjectSyncSettings(projectId, {forceSync, changeUser, projectSyncFrequency}).then(res => {
+      notification.success({ message: res.result });
+      if(forceSync === true) {
+        setDisableSyncNow(true);
+      }
+    }).catch((message) => {
+      notification.error({ message });
+    });
+  }
+
+  return (
+    <>
+      <h1 className="t-main-heading">{i18n("ProjectRemoteSettings.heading")}</h1>
```

Comment:
Can you use the ant design heading

Reply:
Updated in 910177cfbe369c72a9c125601432c54076e13bc2